transit_odp/pipelines/pipelines/dataset_etl/transform.py

```diff
@@ -201,3 +309,97 @@ def get_most_common_districts(stops: pd.DataFrame):
     @staticmethod
     def get_most_common_localities(stops: pd.DataFrame):
         return get_most_common_localities(stops)
+
+    def sync_flexible_zone(self, flexible_stop_points):
+        filtered_stop_points = flexible_stop_points[
+            (flexible_stop_points["bus_stop_type"] == "flexible")
+            & (~flexible_stop_points["naptan_id"].isna())
```

Comment:
Flexible stops which are not in stop point and not n flexible zone will be removed. We need this data even though its not in naptan and flexible zone